aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -190,6 +190,8 @@
    */
   public static final String CLIENT_ASSUME_ROLE_ARN = "client.assume-role.arn";
 
+  public static final String CLIENT_ASSUME_ROLE_TAGS = "client.assume-role.tags";
```

Comment:
can you add some javadoc for this, similar to the other config keys

Reply:
Done. Thanks.